warehouse/manage/views.py

```diff
@@ -558,18 +560,18 @@ def manage_project_roles(project, request, _form_class=CreateRoleForm):
                 .join(Role.user)
                 .filter(Role.role_name == 'Owner', Role.project == project)
             )
-            owner_emails = [owner.user.email for owner in owners]
-            owner_emails.remove(request.user.email)
-
-            send_collaborator_added_email(
-                request,
-                user,
-                request.user,
-                project.name,
-                form.role_name.data,
-                owner_emails
-            )
-
+            owner_emails = [owner.user.email for owner in owners
+                            if owner.user.email]
+
+            if owner_emails:
```

Comment:
Can we move this condition into the `send_collaborator_added_email` function instead?